### PR TITLE
feat: create table expenses on expenses page

### PR DESCRIPTION
**Pull request by Yulatun**
### Type of change:

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [ ] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist:

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible
> If not explain here
- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change
image

#### Screenshots/Video after change

https://user-images.githubusercontent.com/39521158/181479211-7583e917-9bb9-42ac-a725-b78b41071161.mov

update(change Note into icon)
image

update expanses page (I decided to leave alone width )
image

### CODE REVIEW

**Comment by itsnotpolza** [comment]
make edit and delete button closer and maybe hide the note in a tooltip

File: src/pages/Expenses.jsx [hunk 1]
```diff
@@ -1,19 +1,62 @@
-import { Box, Flex, useColorModeValue } from '@chakra-ui/react';
-import i18next from 'i18next';
+import { Box, useColorModeValue } from '@chakra-ui/react';
 
-import { Footer } from '@/components';
+// import i18next from 'i18next';
```
Comment (kalashnikau) [inline src/pages/Expenses.jsx:3]: remove or uncomment

Comment (Yulatun) [reply src/pages/Expenses.jsx:3]: done

File: src/pages/Expenses.jsx [hunk 2]
```diff
@@ -21,22 +20,68 @@ export const FilterTag = ({ text }) => {
...
+    }
+  ];
+  const expensesTable = transactionsTemplate.map((singleTransaction) => {
+    return (
+      <ExpenseItem
+        key={singleTransaction.id}
+        transaction={singleTransaction}
+        onEdit={onEdit}
+        onDelete={onDelete}
+      ></ExpenseItem>
```
Comment (itsnotpolza) [inline src/pages/Expenses.jsx:68]:
```suggestion
      <ExpenseItem
        key={singleTransaction.id}
        transaction={singleTransaction}
        onEdit={onEdit}
        onDelete={onDelete} 
      />```

**Review by PavelTitov16 — DISMISSED** [review]

File: src/pages/Expenses.jsx [hunk 3]
```diff
@@ -21,22 +20,68 @@ export const FilterTag = ({ text }) => {
...
+      <Box bg={bgMain} w="100%">
         <Flex bg={cardBg} direction="column" justify="center" align="center">
           <Box bg={bgMain} w="100%" px={24} py={6}>
             <HStack spacing={4}>
               {['Food', 'Beauty', 'Utilities'].map((name) => (
                 <FilterTag key={name} text={name} />
               ))}
             </HStack>
+
+            <Box>{expensesTable}</Box>
```
Comment (Siqill) [inline src/pages/Expenses.jsx:83]: You can map it here not to create a new variable like it did above ( HStack )

Comment (Yulatun) [reply src/pages/Expenses.jsx]: did it, I was thinking that putting code to variables makes reading it easier

File: src/pages/Expenses.jsx [hunk 4]
```diff
@@ -21,22 +20,67 @@ export const FilterTag = ({ text }) => {
...
+                  transaction={singleTransaction}
+                  onEdit={onEdit}
+                  onDelete={onDelete}
+                />
+              ))}
+            </Box>
           </Box>
-          <main>
-            <h2>{i18next.t('expenses.welcomeMessage')}</h2>
-          </main>
```
Comment (PavelTitov16) [inline src/pages/Expenses.jsx:39]: if we don't need this message anymore maybe delete him in i18n_en.json file?

**Review by itsnotpolza — APPROVED** [review]

**Review by PavelTitov16 — APPROVED** [review]

**Pull request merged**